msos/MixedStack.cs

```diff
@@ -167,17 +175,102 @@ class ThreadInfo
         public uint OSThreadId { get; set; }
         public ClrThread ManagedThread { get; set; }
         public string Detail { get; set; }
-
         public bool IsManagedThread { get { return ManagedThread != null; } }
     }
 
+    class UnifiedThread
+    {
+        public UnifiedThread(ThreadInfo info)
+        {
+            IsManagedThread = info.IsManagedThread;
+            Index = info.Index;
+            EngineThreadId = info.EngineThreadId;
+            OSThreadId = info.OSThreadId;
+            Detail = info.Detail;
+            Info = info;
+        }
+
+        public UnifiedThread(uint owningThreadId)
```

Comment:
Why is this called `owningThreadId`? What does it mean for a thread to be owned by a thread? Should this just be called `osThreadId`?